python/ffsim/gates/basic_gates.py

```diff
@@ -570,3 +570,53 @@ def apply_fsim_gate(
         vec, -phi, target_orbs, norb=norb, nelec=nelec, spin=spin, copy=False
     )
     return vec
+
+
+def apply_fswap_gate(
+    vec: np.ndarray,
+    target_orbs: tuple[int, int],
+    norb: int,
+    nelec: int | tuple[int, int],
+    spin: Spin = Spin.ALPHA_AND_BETA,
+    *,
+    copy: bool = True,
+) -> np.adarray:
+    r"""Apply a fSWAP gate 
+
+        .. math::
+
+        \begin{pmatrix}
+            1 & 0 & 0 & 0 \\
+            0 & 0 & 1 & 0 \\
+            0 & 1 & 0 & 0 \\
+            0 & 0 & 0 & -1 \\
+        \end{pmatrix}
```

Comment:
You need to clarify that this matrix is only valid for neighboring qubits under the Jordan-Wigner transform. You can copy the text from here: https://github.com/qiskit-community/ffsim/blob/a2891d6c641a3ce3fe5ef2acbbba5b23db4d7977/python/ffsim/gates/basic_gates.py#L454-L464

Also I think the matrix needs to be indented. Please make sure that it looks good in the built documentation.